thingsvision/utils/alignment/transform.py

```diff
@@ -0,0 +1,52 @@
+__all__ = ["Transform"]
+
+from typing import Any
+
+import numpy as np
+import requests
+
+Array = np.ndarray
+
+gLocal_URL = "https://raw.githubusercontent.com/LukasMut/gLocal/main/transforms/"
+
+
+class Transform:
+    def __init__(
+        self, model_name: str, module_name: str, alignment_type: str = "gLocal"
+    ) -> None:
+        self.model_name = model_name
+        self.module_name = module_name
+        if alignment_type == "gLocal":
+            self.transform = self._load_transform()
+        else:
+            raise NotImplementedError(
+                f"\nRepresentational alignment of type: {alignment_type} is not yet implemented.\nChange to gLocal!\n"
+            )
+
+    def _load_transform(self) -> Any:
+        url = os.path.join(
```

Comment:
Maybe this URL could be moved into the __init__, where the alignment_type is processed, since it seems gLocal-specific.

Reply:
Done.